### PR TITLE
Adds contributing guide and CoD

### PR DESCRIPTION
**Pull request by kipparker**
I think it's worth adding a contributing guide, I've created this from the boiler plate at https://contributing.md (seems very widely used), editing it for brevity and adding our details. The short version is:

- Enhancements / improvement / bugs as GitHub issues
- Ask or help in the Slack channel
- Security issues via email

Alternatively we could add details of this to the README.md. 

I've included the [code of conduct](https://github.com/icebreakerone/perseus-demo-energy/pull/38/files#diff-ffdbe3a1e7ee93cacfc080b6c635ccf3a8f6b0f00f2fb884f78c6b5f9dac8fd2) from contributing.md without edits, but we might want to leave that out until we have the time to properly review it and invite input from others, or there may be existing content from ib1 code of conduct / membership terms documents that we ought to include

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+This project and everyone participating in it is governed by the
+[Perseus Demo Energy Code of Conduct](https://github.com/icebreakerone/perseus-demo-energyblob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior
+to [perseus@ib1.org](mailto:perseus@ib1.org).
+
+## Asking questions
+
+### Perseus participants
+
+A slack channel is available for participants building prototypes for the Perseus trust framework pilot, and is the best place to ask questions. If you are not part of the slack channel and wish to have access, please email [perseus@ib1.org](mailto:perseus@ib1.org)
```
Comment (kipparker) [inline CONTRIBUTING.md:26]: Participants are guided to the Slack channel to ask questions

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### Other users
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/icebreakerone/perseus-demo-energy/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
```
Comment (kipparker) [inline CONTRIBUTING.md:32]: Other users are guided to github issues to ask questions

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- Collect information about the bug:
+- Stack trace (Traceback)
+- OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+- Version of the interpreter, compiler, SDK, runtime environment, package manager, depending on what seems relevant.
+- Possibly your input and the output
+
+#### How Do I Submit a Good Bug Report?
+
+> Please do not report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead sensitive bugs should be sent by email to perseus@ib1.org
+
```
Comment (kipparker) [inline CONTRIBUTING.md:68]: Note that security issues should not be raised as issues, but sent as emails

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/icebreakerone/perseus-demo-energy/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why
+- Provide links to code, if applicable
+
+### Improving The Documentation
+
+Feedback on our documentation is always welcome. If you see something that is wrong, missing, or could be explained better, you can help us by creating an issue, or by creating a pull request with the suggested changes.
```
Comment (kipparker) [inline CONTRIBUTING.md:100]: Not sure if we should put something about other channels for comments on documentation? Thinking that anyone reading this is probably a github user, so will be able to deal with raising issues / pull requests

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at .
```
Comment (bensummers) [inline CODE_OF_CONDUCT.md:59]: Missing email address or means of communication.

**Review by bensummers — CHANGES_REQUESTED** [review]
Just need to fill in a missing bit of data. Text feels quite standard, as you'd expect from a generator.

**Pull request merged**